### PR TITLE
Fix LGTM warnings

### PR DESCRIPTION
**Pull request by nritsche**

### CODE REVIEW

File: lib/stages/RfiFrameDrop.hpp [hunk 1]
```diff
@@ -3,8 +3,8 @@
 @brief Drops GPU frames when they are contaminated with RFI
 - RfiFrameDrop : public kotekan::Stage
 *****************************************/
-#ifndef VALVE_HPP
-#define VALVE_HPP
+#ifndef RFI_FRAME_DROP_HPP
```
Comment (jrs65) [inline lib/stages/RfiFrameDrop.hpp:6]: Ha oops. Mea cupla.

File: python/kotekan/_version.py [hunk 2]
```diff
@@ -524,7 +524,7 @@ def get_versions():
         # versionfile_source is the relative path from the top of the source
         # tree (where the .git directory might live) to this file. Invert
         # this to find the root from __file__.
-        for i in cfg.versionfile_source.split("/"):
+        for _ in cfg.versionfile_source.split("/"):
```
Comment (jrs65) [inline python/kotekan/_version.py:527]: I'd be tempted to ignore this one. As this file is produced by versioneer it'll make it annoying to see what's going on if we ever update versioneer and re-gen this file.

File: python/requirements.txt [hunk 3]
```diff
@@ -4,6 +4,7 @@ click
 h5py
 future
 requests
+shutil
```
Comment (jrs65) [inline python/requirements.txt:7]: Why? It's a built in module.

**Review by jrs65 — CHANGES_REQUESTED** [review]
Most of the LGTM suggestions seem to be pretty good, although there's a few slightly funky ones.

**Comment by nritsche** [comment]
Thanks @jrs65 ! Done.

**Review by jrs65 — APPROVED** [review]

File: python/kotekan/_version.py (hunk 2 above)
Comment (nritsche) [reply python/kotekan/_version.py]: I think we blackened that file btw :laughing:

**Review by andrerenard — APPROVED** [review]

**Pull request merged**